spec/features/user_can_update_qty_of_item_in_cart_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.